### PR TITLE
fixed currency and spinner bugs

### PR DESCRIPTION
**Pull request by otigreros**
Fix the currency error in some cases don't show the value
fix the spinner bug that make the droplets don't disappear when the view is swiped

### CODE REVIEW

File: app/src/main/java/co/condorlabs/customcomponents/helper/masks/PriceTextWatcherMask.kt [hunk 1]
```diff
@@ -38,13 +41,18 @@ class PriceTextWatcherMask(private val receiver: EditText) : TextWatcherAdapter(
...
+        setSelectionAndListener()
     }
 
     override fun afterTextChanged(s: Editable?) {
         val text = s?.toString() ?: return
 
         receiver.removeTextChangedListener(this)
 
+
+
```
Comment (EAlexisDuque) [inline app/src/main/java/co/condorlabs/customcomponents/helper/masks/PriceTextWatcherMask.kt]: remove spaces

File: app/src/main/java/co/condorlabs/customcomponents/customedittext/EditTextCurrencyField.kt [hunk 2]
```diff
@@ -48,7 +49,16 @@ class EditTextCurrencyField(context: Context, attrs: AttributeSet) : BaseEditTex
...
+        if (focused) {
+            if (editText?.text.isNullOrEmpty()) {
+                editText?.setText(DOLLAR_SYMBOL)
+            }
+        }
+    }
+
     private fun isFieldEmpty(text: String?): Boolean {
-        return text == DOLLAR_SYMBOL
+        return text == DOLLAR_SYMBOL || text == EMPTY
```
Comment (feguerrero) [inline app/src/main/java/co/condorlabs/customcomponents/customedittext/EditTextCurrencyField.kt]: Then rename the function: `isNotFilledWithDigits`

File: app/src/main/java/co/condorlabs/customcomponents/helper/masks/PriceTextWatcherMask.kt (hunk 1 above)
Comment (feguerrero) [inline app/src/main/java/co/condorlabs/customcomponents/helper/masks/PriceTextWatcherMask.kt]: Remove those blank lines

File: app/src/main/java/co/condorlabs/customcomponents/helper/masks/PriceTextWatcherMask.kt [hunk 3]
```diff
@@ -88,10 +113,11 @@ class PriceTextWatcherMask(private val receiver: EditText) : TextWatcherAdapter(
                 return
             }
 
-            if (text.last() == ZERO_CHARACTER) {
+            if (text.last() == ZERO_CHARACTER && text != ZERO_CHARACTER.toString()) {
```
Comment (feguerrero) [inline app/src/main/java/co/condorlabs/customcomponents/helper/masks/PriceTextWatcherMask.kt]: Better reuse `isZeroLastDecimal`()

File: app/src/main/java/co/condorlabs/customcomponents/customedittext/EditTextCurrencyField.kt (hunk 2 above)
Comment (otigreros) [reply app/src/main/java/co/condorlabs/customcomponents/customedittext/EditTextCurrencyField.kt]: done

File: app/src/main/java/co/condorlabs/customcomponents/helper/masks/PriceTextWatcherMask.kt (hunk 1 above)
Comment (otigreros) [reply app/src/main/java/co/condorlabs/customcomponents/helper/masks/PriceTextWatcherMask.kt]: done

File: app/src/main/java/co/condorlabs/customcomponents/helper/masks/PriceTextWatcherMask.kt (hunk 3 above)
Comment (otigreros) [reply app/src/main/java/co/condorlabs/customcomponents/helper/masks/PriceTextWatcherMask.kt]: done

**Pull request merged**